### PR TITLE
Hailey

### PR DESCRIPTION
**Pull request by Haileyang**
Please review

### CODE REVIEW

File: invitation/package.json [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+			"not op_mini all"
+		],
+		"development": [
+			"last 1 chrome version",
+			"last 1 firefox version",
+			"last 1 safari version"
+		]
+	},
+	"devDependencies": {
+		"react-scripts": "^5.0.1"
```
Comment (HanKyungSung) [inline invitation/package.json:45]: why 4 indentation .. ?

Comment (HanKyungSung) [reply invitation/package.json:45]: 왜 why?

Comment (Haileyang) [reply invitation/package.json:45]: :) cuz u want it ?

**Review by HanKyungSung — APPROVED** [review]
So far so g00d

**Pull request merged**